homeassistant/components/accuweather/weather.py

```diff
@@ -73,6 +73,12 @@ def __init__(
             identifiers={(DOMAIN, coordinator.location_key)},
             manufacturer=MANUFACTURER,
             name=NAME,
+            # You don't need to provide specific details for the URL,
+            # so passing in _ characters is fine if the location key
+            # is correct
+            configuration_url="http://accuweather.com/en/"
```

Comment:
It's much easier to read if we wrap the string in parentheses. Black will then format the string parts on new lines indented.